src/components/LocalIdentityModal/LocalIdentityModal.js

```diff
@@ -87,7 +81,7 @@ const Modal = ({ address, label, onCancel, onSave }) => {
       <Wrap>
```

Comment:
We should remove the extra padding on `Wrap` since the `Modal` applies it by default now.